app/src/main/java/com/android/sample/ui/calendar/DayCalendarScreen.kt

```diff
@@ -65,47 +81,53 @@ fun DayCalendarScreen(
       dailyWorkouts
           .map {
             when (it) {
-              is BodyWeightWorkout -> ColoredWorkout(it, BodyWeightTag, WorkoutType.BODY_WEIGHT)
-              is YogaWorkout -> ColoredWorkout(it, YogaTag, WorkoutType.YOGA)
-              else -> ColoredWorkout(it, RunningTag, WorkoutType.RUNNING)
+              is BodyWeightWorkout -> DayColoredWorkout(it, LegendBodyweight, bodyworkoutViewModel)
+              is YogaWorkout -> DayColoredWorkout(it, LegendYoga, yogaworkoutViewModel)
+              else ->
+                  DayColoredWorkout(
+                      it,
+                      LegendRunning,
+                      bodyworkoutViewModel) // Temps until we got some saved running workouts
             }
           }
           .sortedBy { it.workout.date.minute }
 
-  Scaffold(
-      topBar = { TopBar(navigationActions, R.string.calendar_title) },
-      bottomBar = { BottomBar(navigationActions) }) { padding ->
-        Column(modifier = Modifier.fillMaxSize().padding(padding)) {
-          Legend()
+  Scaffold(topBar = { TopBar(navigationActions, R.string.calendar_title) }) { padding ->
+    Column(modifier = Modifier.fillMaxSize().padding(padding)) {
+      Legend()
 
-          Divider(
-              color = Color.LightGray,
-              thickness = 1.dp,
-              modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp))
+      Divider(
+          color = Color.LightGray,
+          thickness = 1.dp,
+          modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp))
 
-          Text(
-              text =
-                  selectedDate?.toJavaLocalDate()?.format(DateTimeFormatter.ofPattern("d MMMM"))
-                      ?: "",
-              style = MaterialTheme.typography.headlineMedium,
-              fontWeight = FontWeight.Bold,
-              modifier =
-                  Modifier.padding(top = 8.dp, start = 16.dp, bottom = 16.dp).testTag("Date"))
+      Text(
+          text =
+              selectedDate?.toJavaLocalDate()?.format(DateTimeFormatter.ofPattern("dd MMMM")) ?: "",
+          fontFamily = OpenSans,
+          fontWeight = FontWeight.SemiBold,
+          fontSize = 45.sp,
```

Comment:
If you plan on reusing this same font size on another screen later it would probably be better to add it to the FontSizes class for more consistency